package-lock.json

```diff
@@ -0,0 +1,5817 @@
+{
```

Comment:
I noticed we added a `package-lock` in this PR, should we continue with Yarn or go only with npm? If we decide to use `npm`, we need to delete the `yarn.lock` file.

Reply:
Per the Slack conversation, I've removed the `yarn.lock` file and I've also changed the readme to reflect NPM install and run instructions.